security/.htaccess

```diff
@@ -38,6 +38,12 @@ RewriteRule ^suites/aes-2019/v1$ https://digitalbazaar.github.io/aes-key-wrappin
 RewriteRule ^suites/hmac-2019$ https://digitalbazaar.github.io/sha256-hmac-key-2019-context [R=302,L]
 RewriteRule ^suites/hmac-2019/v1$ https://digitalbazaar.github.io/sha256-hmac-key-2019-context/contexts/sha256-hmac-2019-v1.jsonld [R=302,L]
 
+
+# http://w3id.org/security/suites/multibase-2021
+
+RewriteRule ^suites/multibase-2021$ https://ns.did.ai/suites/multibase-2021 [R=302,L]
+RewriteRule ^suites/multibase-2021/v1$ https://ns.did.ai/suites/multibase-2021/v1 [R=302,L]
```

Comment:
Multibase isn't a cryptography suite anymore than base64 is. The name of the suite is misleading... at the very least, change it to "multikey"... but do that knowing that you're going to have to resolve this with the IPFS community... who have seemed to abandon the concept of multikey a while ago: https://github.com/ipfs/specs/issues/58

You're resurrecting something that's been dead for a while now... I have no issue with you doing that, except that I hope you will do all the due diligence necessary to build bridges between communities when you do this.

Again, I completely disagree with trying to bring the brokenness of JWK into the LD Security community, but it's not illegal to do so.

Please rename to multikey-2021 and I won't stand in the way.